go-controller/pkg/node/secondary_node_network_controller_test.go

```diff
@@ -60,13 +63,22 @@ var _ = Describe("SecondaryNodeNetworkController", func() {
 		Expect(err).NotTo(HaveOccurred())
 		err = testNS.Do(func(ns.NetNS) error {
 			defer GinkgoRecover()
+			ovntest.AddLink(gatewayInterface)
+			ovntest.AddLink(gatewayBridge)
 			ovntest.AddLink(mgtPort)
 			return nil
 		})
 		Expect(err).NotTo(HaveOccurred())
 		wg = &sync.WaitGroup{}
 		stopCh = make(chan struct{})
-		vrf = vrfmanager.NewController()
+		routeManager := routemanager.NewController()
+		wg.Add(1)
+		go testNS.Do(func(netNS ns.NetNS) error {
+			defer wg.Done()
+			routeManager.Run(stopCh, 4*time.Minute)
```

Comment:
recommend lowering sync value so we can increase the odds of capturing `sync` errors

Reply:
getting this down to 2mins... is that fine?